Berlin/BerOrOct555.xml

```diff
@@ -0,0 +1,218 @@
+                  <collection>Ms. orient. oct.</collection>
+
+                  <idno facs="https://content.staatsbibliothek-berlin.de/dc/863181112/manifest">SBP, Ms. orient. oct. 555</idno>
+               </msIdentifier>
+
+               <additional>
+                  <adminInfo>
+                  <recordHist>
+                  <source>
+                    <listBibl type="secondary">
+                    <bibl>
+                    <ptr target="bm:HammerschmidtJaeger1968Illuminiert"/>
+                      <citedRange unit="page">50-54</citedRange>
+                    </bibl>
+                    </listBibl>
+                    </source>
+                  </recordHist>
+                  </adminInfo>
+                </additional>
+
+            <msPart xml:id="p1">
+               <msIdentifier> <idno>SBP, Ms. orient. oct. 555, p1</idno>
+             </msIdentifier>
+
+               <msContents>
+                 <msItem xml:id="p1_i1">
+                     <title ref="LIT1119DossierQMQ"/>
+                 <msItem xml:id="p1_i1.1">
+                     <locus from="6ra" to="94rb"/>
+                     <title type="complete" ref="LIT1451GadlaG"/>
+                     <textLang mainLang="gez"/>
+                     <incipit xml:lang="gez"> <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወ</hi>መንፈስ፡ ቅዱስ፡ <hi rend="rubric">፩</hi>አም<hi rend="rubric">ላክ፡ በስመ፡ እግዚአ</hi>ብሔር፡ አብ፡
+                     ዘኢይትዌ<hi rend="rubric">ለጥ፡ እምህላዌሁ።</hi> ወበስሙ፡ ለወልድ፡ እግ<hi rend="rubric">ዚአብሔር፡ ዘኢይተፈ</hi>ለጥ፡ እምህላዌሁ፡ ለአ<hi rend="rubric">ቡሁ።</hi> </incipit>
+                     <note>The manuscript is ms M in Marrassini's 2003 edition of the text.</note>
+                 </msItem>
+                 <msItem xml:id="p1_i1.2">
+                     <locus from="94va" to="118a"/>
+                     <title type="complete" ref="LIT3977Taammer"/>
+                     <textLang mainLang="gez"/>
+                      <incipit xml:lang="gez"> <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi> ወመንፈስ፡ ቅዱስ፡ ፩፡ <hi rend="rubric">አምላክ። ንጽሐፈ፡ ዜ</hi>ናሁ፡ ወ<del/>፡
+                      ወተአ<hi rend="rubric">ምራቲሁ፡ ለብፁዕ፡ ወ</hi>ቅዱስ፡ አቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡</incipit>
+                 </msItem>
+                 </msItem>
+               </msContents>
+
+               <physDesc>
+
+                 <objectDesc form="Codex">
+                    <supportDesc>
+                       <support>
+                          <material key="parchment"/>
+                       </support>
+                     <extent>
+                       <measure unit="leaf">120</measure>
+                        <dimensions type="outer" unit="mm">
+                           <height>150</height>
+                           <width>130</width>
+                           <depth>70</depth>
+                        </dimensions>
+                      </extent>
+
+                      <foliation>
+                         <list>
+                            <item xml:id="f.1r">
```

Comment:
I really don't think that the list element is necessary if there is only one foliation. I would simply write
<foliation> <locus from="1r" to="120r"/>Foliation in pencil in the upper right corner.</foliation>.